### DISCUSSION TITLE
Home Assistant 2024.8

### DISCUSSION THREAD

**Discussion by stevetrease** [nathanmarlor/foxess_modbus#657] [General]
Hello, 

Before I dive in and upgrade, has anyone confirmed the integration works with this week's release of Home Assistant 2024.8?

Cheers,

Steve

**Comment by pgmelo1** [comment]
Hi,

Worked well for me, everything working with no problems.

**Reply by stevetrease** [reply]
I upgraded yesterday and can confirm that I see no problems.
